### PR TITLE
Update commands to run on EC2 instances

### PR DESCRIPTION
**Pull request by sabau**
**Description:**
- Align commands with reality

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: .aws/faucet.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-AMOUNT=100stake
+AMOUNTS=5stake
+#AMOUNTP=5photino
```
Comment (fedekunze) [inline .aws/faucet.sh]: uncomment or delete ?

File: .aws/faucet.sh [hunk 2]
```diff
@@ -17,7 +19,8 @@ do
 
             # Just in case we were running this command with rest-server switched on, get again the address
             ADDRESS=$(./gaiacli keys show ${ACCOUNT} --home . --address)
-            echo ${PASSWORD} | ./gaiacli tx send ${DESTINATION} ${AMOUNT} --home . --from ${ADDRESS} --chain-id=${NETWORK}
+            echo ${PASSWORD} | ./gaiacli tx send ${DESTINATION} ${AMOUNTS} --home . --from ${ADDRESS} --chain-id=${NETWORK}
+#            echo ${PASSWORD} | ./gaiacli tx send ${DESTINATION} ${AMOUNTP} --home . --from ${ADDRESS} --chain-id=${NETWORK}
```
Comment (fedekunze) [inline .aws/faucet.sh]: ditto

File: .aws/faucet.sh (hunk 1 above)
Comment (sabau) [reply .aws/faucet.sh]: I wanted to keep it and uncomment for quick tests on the instance, probably when we will have a quick deploy we won't need this anymore

File: .aws/faucet.sh (hunk 2 above)
Comment (sabau) [reply .aws/faucet.sh]: datto

File: .aws/faucet.sh [hunk 3]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-AMOUNT=100stake
+AMOUNTS=5stake
```
Comment (faboweb) [inline .aws/faucet.sh]: like said yesterday: this will display as `0.0000005` stake. I vote for increasing all values by `10e6`

Comment (jbibla) [reply .aws/faucet.sh]: yep good call!

File: .aws/faucet.sh (hunk 2 above)
Comment (sabau) [reply .aws/faucet.sh]: I re-enalbed those, apparently the drainig of money always happened at `stakes`

File: .aws/faucet.sh (hunk 3 above)
Comment (sabau) [reply .aws/faucet.sh]: pumped all money upmillion

File: .aws/faucet.sh (hunk 1 above)
Comment (sabau) [reply .aws/faucet.sh]: uncommented in the end is healthier to check also on different denoms

**Pull request merged**